<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vasile-PC" ignored="0" name="Failure Test" tests="1" failures="1" timestamp="2020-06-20T13:37:45 EEST" time="0.002" errors="0">
  <testcase name="doLogin" time="0.001" classname="testcases.TestFailure">
    <failure type="java.lang.AssertionError" message="Failing the login test">
      <![CDATA[java.lang.AssertionError: Failing the login test
at testcases.TestFailure.doLogin(TestFailure.java:11)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- Failure Test -->
